packages/cursorless-everywhere-talon/scripts/esbuild.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+esbuild \
+  --outfile=out/talon.js \
+  --platform=neutral \
+  --format=esm \
+  --main-fields=main,module \
+  --conditions=cursorless:bundler \
+  --bundle \
+  --sourcemap \
+  --external:talon \
+  "$@"
+
+# Talon javascript files needs to start with an import from Talon before any other code
```

Comment:
Getting the impression this won't be fixed before we ship? Shall we file an issue / add a FIXME?